containerd-shim-v2/wait.go

```diff
@@ -151,6 +153,26 @@ func watchOOMEvents(ctx context.Context, s *service) {
 				continue
 			}
 
+			// write oom file for CRI-O
+			if c, ok := s.containers[containerID]; ok && oci.IsCRIOContainerManager(c.spec) {
```

Comment:
@liubin I am a bit skeptical about adding code checking for specific container manager. Can we simply create this file irrespective of what the upper stack looks like?
cc @bergwolf